tools/setup/emoji/emoji_setup_utils.py

```diff
@@ -15,115 +15,6 @@
-blacklisted_names = frozenset([
-    # would be chosen by words_supersets or superstrings
-    'football', # american_football
-    'post_office', # european_post_office (there's also a japanese_post_office)
-    'castle', # european_castle (there's also a japanese_castle)
-    'chart', # chart_increasing_with_yen (should rename chart_increasing to chart)
-    'loop', # double_curly_loop (should rename curly_loop to loop)
-    'massage', # face_massage
-    'bulb', # light_bulb
-    'barber', # barber_pole
-    'mens', # mens_room
-    'womens', # womens_room
-    'knife', # kitchen_knife (hocho also maps here)
-    'notes', # musical_notes
-    'beetle', # lady_beetle
-    'ab', # ab_button (due to keeping a_button, due to the one_lettered() rule)
-    'headphone', # headphones
-    'mega', # megaphone
-    'ski', # skis
-    'high_heel', # high_heeled_shoe (so that it shows up when searching for shoe)
-    # less confident about the following
-    'dolls', # japanese_dolls
-    'moon', # waxing_gibbous_moon (should rename crescent_moon to moon)
-    'clapper', # clapper_board
-    'traffic_light', # horizontal_traffic_light (there's also a vertical_traffic_light)
-    'lantern',
-    'red_paper_lantern', # izakaya_lantern (in the future we should make sure
-                         # red_paper_lantern finds this)
-
-    # would be chosen by longer
-    'down_button', # arrow_down_small, I think to match the other arrow_*
-                   # names. Matching what github and slack do.
-    'running_shoe', # athletic_shoe, both github and slack agree here.
-    'running', # runner. slack has both, github has running_man and running_woman, but not runner
-    'o2', # o_button
-    'star2', # glowing_star
-    'bright', # high_brightness, to match low_brightness, what github/slack do
-    'dim_button', # low_brightness, copying github/slack
-    'stars', # shooting_star. disagrees with github, slack, and emojione, but this seems better
-    'nail_care', # nail_polish. Also disagrees github/slack/emojione, is nail_polish mostly an
-                 # american thing?
-    'busstop', # bus_stop
-    'tophat', # top_hat
-    'old_woman', # older_woman, following github/slack/emojione on these
-    'old_man', # older_man
-    'blue_car', # recreational_vehicle
-    'litter_in_bin_sign', # put_litter_in_its_place
-    'moai', # moyai based on github/slack
-    'fuelpump', # fuel_pump
-
-    # names not otherwise excluded by our heuristics
-    'left_arrow', # arrow_left, to match other arrow_* shortnames
-    'right_arrow', # arrow_right
-    'up_arrow', # arrow_up
-    'down_arrow', # arrow_down
-    'chequered_flag', # checkered_flag
-    'e_mail', # e-mail
-    'non_potable_water', # non-potable_water
-    'flipper', # dolphin
-])
-
 # A handcoded list of popular emoji names. This is useful for cases
 # where the two names are very different, users might reasonably type
 # either name and be surprised when they can't find the relevant emoji.
```

Comment:
I see, I didn't realize we already had the ability for multiple names in the autocomplete. In that case I think we should just include all the short_codes programatically. And maybe blacklist 'shit'?

In case you're curious, here is the list:

[(u'phone', [u'phone', u'telephone']),
 (u'boat', [u'boat', u'sailboat']),
 (u'email', [u'email', u'envelope']),
 (u'hand', [u'hand', u'raised_hand']),
 (u'exclamation', [u'exclamation', u'heavy_exclamation_mark']),
 (u'moon', [u'moon', u'waxing_gibbous_moon']),
 (u'mostly_sunny', [u'mostly_sunny', u'sun_small_cloud']),
 (u'barely_sunny', [u'barely_sunny', u'sun_behind_cloud']),
 (u'partly_sunny_rain', [u'partly_sunny_rain', u'sun_behind_rain_cloud']),
 (u'lightning', [u'lightning', u'lightning_cloud']),
 (u'tornado', [u'tornado', u'tornado_cloud']),
 (u'runner', [u'runner', u'running']),
 (u'izakaya_lantern', [u'izakaya_lantern', u'lantern']),
 (u'bee', [u'bee', u'honeybee']),
 (u'dolphin', [u'dolphin', u'flipper']),
 (u'feet', [u'feet', u'paw_prints']),
 (u'facepunch', [u'facepunch', u'punch']),
 (u'+1', [u'+1', u'thumbsup']),
 (u'-1', [u'-1', u'thumbsdown']),
 (u'shirt', [u'shirt', u'tshirt']),
 (u'mans_shoe', [u'mans_shoe', u'shoe']),
 (u'family', [u'family', u'man-woman-boy']),
 (u'couple', [u'couple', u'man_and_woman_holding_hands']),
 (u'boom', [u'boom', u'collision']),
 (u'hankey', [u'hankey', u'poop', u'shit']),
 (u'book', [u'book', u'open_book']),
 (u'memo', [u'memo', u'pencil']),
 (u'hocho', [u'hocho', u'knife']),
 (u'middle_finger',
  [u'middle_finger', u'reversed_hand_with_middle_finger_extended']),
 (u'laughing', [u'laughing', u'satisfied']),
 (u'car', [u'car', u'red_car']),
 (u'the_horns', [u'the_horns', u'sign_of_the_horns']),
 (u'flag-cn', [u'flag-cn', u'cn']),
 (u'flag-de', [u'flag-de', u'de']),
 (u'flag-es', [u'flag-es', u'es']),
 (u'flag-fr', [u'flag-fr', u'fr']),
 (u'flag-gb', [u'flag-gb', u'gb', u'uk']),
 (u'flag-it', [u'flag-it', u'it']),
 (u'flag-jp', [u'flag-jp', u'jp']),
 (u'flag-kr', [u'flag-kr', u'kr']),
 (u'flag-ru', [u'flag-ru', u'ru']),
 (u'flag-us', [u'flag-us', u'us'])]